tests/unit/unittests.sh

```diff
@@ -215,25 +215,27 @@ for MODE in debug ndebug; do
 				grep 'Test OK' ${TEST_OUT_DIR}/ewiseapply_large_${MODE}_${BACKEND}_${P}_${T} || echo "Test FAILED"
 				echo " "
 
-				echo ">>>      [x]           [ ]       Testing grb::eWiseApply using + on matrices"
-				$runner ${TEST_BIN_DIR}/eWiseApply_matrix_${MODE}_${BACKEND} &> ${TEST_OUT_DIR}/eWiseApply_matrix_${MODE}_${BACKEND}_${P}_${T}
-				head -1 ${TEST_OUT_DIR}/eWiseApply_matrix_${MODE}_${BACKEND}_${P}_${T}
-				grep 'Test OK' ${TEST_OUT_DIR}/eWiseApply_matrix_${MODE}_${BACKEND}_${P}_${T} || echo "Test FAILED"
-				echo " "
+				if [ "$BACKEND" != "nonblocking" ]; then
```

Comment:
There are multiple such if-statements. Please group all tests that are currently not supported by the nonblocking backend together so that there is only one such if-statement, or, if it is not possible due to other such clauses, a minimum number of such if-statements.